modeshape-jcr/src/main/java/org/modeshape/jcr/shell/LocalSession.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * ModeShape (http://www.modeshape.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.modeshape.jcr.shell;
+
+import javax.jcr.Session;
+
+/**
+ *
+ * @author kulikov
+ */
+public class LocalSession {
```

Comment:
is this wrapper required ? it doesn't seem to add anything else on top of a `javax.jcr.Session`